aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
- make sure you have a git remote pointing to `https://github.com/Turbo87/aerofiles.git`
   - if you cloned this then the remote should be there automatically and called `origin`
   - if you don't have one use something like `git remote add upstream https://github.com/Turbo87/aerofiles.git`
- `git fetch upstream` (or whatever that remote above is called)
- `git checkout -b seeyou-tasks upstream/master` to create a new branch called `seeyou-tasks` at the same commit that `upstream/master` points to